src/constants.ts

```diff
@@ -2,6 +2,8 @@ import config from "../config.json";
 
 export const PLUGIN_NAME = "Starknet";
 export const ABI_SUFFIX = "_abi.json";
+export const SIERRA_SUFFIX = ".casm";
```

Comment:
I think this is not Sierra suffix. `casm` stands for cairo-assembly

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/328/commits/961412836194e10632588c5dbbd65b6d4c3e28fc